conmon/conmon.c

```diff
@@ -733,6 +801,12 @@ int main(int argc, char *argv[])
 						nwarn("write_k8s_log failed");
 						goto out;
 					}
+
+					if (conn_sock > 0) {
```

Comment:
I was thinking of handling multiple in a follow-on PR as most use cases are handled by one client.